agent/handlers/gql/schema.go

```diff
@@ -314,6 +339,70 @@ func containerARNResolver(p graphql.ResolveParams) (interface{}, error) {
 	return nil, nil
 }
 
+func containerStatsResolver(state dockerstate.TaskEngineState, statsEngine stats.Engine) func(p graphql.ResolveParams) (interface{}, error) {
+	return func(p graphql.ResolveParams) (interface{}, error) {
+		dockerContainer, ok := p.Source.(*apicontainer.DockerContainer)
+		if !ok {
+			return nil, nil
+		}
+		task, ok := state.TaskByID(dockerContainer.DockerID)
+		if !ok {
+			return nil, errors.Errorf("Unable to find task for container '%s'", dockerContainer.DockerID)
+		}
+
+		dockerStats, network_rate_stats, err := statsEngine.ContainerDockerStats(task.Arn, dockerContainer.DockerID)
+		if err != nil {
+			return nil, err
+		}
+
+		return v4.StatsResponse{
+			StatsJSON:          dockerStats,
+			Network_rate_stats: network_rate_stats,
+		}, nil
+	}
+}
+
+func containerNetworksResolver(state dockerstate.TaskEngineState) func(p graphql.ResolveParams) (interface{}, error) {
+	return func(p graphql.ResolveParams) (interface{}, error) {
+		if dockerContainer, ok := p.Source.(*apicontainer.DockerContainer); ok {
+			// Pulled containers do not have a network
+			// TODO: Find a cleaner way to have pulled containers return nil
+			if dockerContainer.Container.KnownStatusUnsafe.String() == "PULLED" {
+				return nil, nil
+			}
+			task, ok := state.TaskByID(dockerContainer.DockerID)
+			if !ok {
+				return nil, errors.Errorf("Unable to find task for container '%s'", dockerContainer.DockerID)
+			}
+			eni := task.GetPrimaryENI()
+			var resp []v4.Network
+			if eni != nil { // Set Network to AWSVPC
```

Comment:
This is a bit confusing. What is being "set" here?

Also, shouldn't this reuse the existing v4 network logic? https://github.com/aws/amazon-ecs-agent/blob/master/agent/handlers/v4/response.go#L126-L129